README.md

```diff
@@ -58,8 +58,9 @@ After installation you can directly start using it. You can do this by either us
 | Show all bookmarks (toggle)                     | `ma`        | `:BookmarkShowAll`           |
 | Clear bookmarks in current buffer only          | `mc`        | `:BookmarkClear`             |
 | Clear bookmarks in all buffers                  | `mx`        | `:BookmarkClearAll`          |
-| Move up bookmark at current line                | `mkk`       | `:BookmarkMoveUp`            |
-| Move down bookmark at current line              | `mjj`       | `:BookmarkMoveDown`          |
+| Move up bookmark at current line                | `[count]mkk`| `:BookmarkMoveUp`            |
+| Move down bookmark at current line              | `[count]mjj`| `:BookmarkMoveDown`          |
+| Move bookmark at current line to another line   |             | `:BookmarkMoveToLine <LINE>` |
```

Comment:
Couldn't we have `BookmarkMoveToLine` and `BookmarkModify` with explicit arguments that call under the hood a (private) function that does what `BookmarkModify` is currently doing. This way the (public) api is clear and the logic would not be duplicated.